<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>600</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#widget{
background-color: #95a5a6;
}
</string>
   </property>
   <widget class="QLabel" name="productImage">
    <property name="geometry">
     <rect>
      <x>335</x>
      <y>200</y>
      <width>140</width>
      <height>140</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
border-right:1px solid #2ecc71;
border-bottom:1px solid #2ecc71;
background-color: rgb(44, 62, 80);
color: rgb(241, 196, 15);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="productBrand">
    <property name="geometry">
     <rect>
      <x>215</x>
      <y>200</y>
      <width>120</width>
      <height>48</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Comic Sans MS&quot;;
border:none;
border-left:1px solid #2ecc71;
border-right:1px solid #2ecc71;
border-bottom:1px solid #2ecc71;
background-color: rgb(44, 62, 80);
color: rgb(241, 196, 15);
</string>
    </property>
    <property name="text">
     <string>Monster</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="productRatingCount">
    <property name="geometry">
     <rect>
      <x>215</x>
      <y>294</y>
      <width>120</width>
      <height>46</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Comic Sans MS&quot;;
border:1px solid #2ecc71;
background-color: rgb(44, 62, 80);
color: rgb(241, 196, 15);

</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="productRating">
    <property name="geometry">
     <rect>
      <x>215</x>
      <y>248</y>
      <width>120</width>
      <height>46</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Comic Sans MS&quot;;
border:none;
border-left:1px solid #2ecc71;
border-right:1px solid #2ecc71;
background-color: rgb(44, 62, 80);
color: rgb(241, 196, 15);

</string>
    </property>
    <property name="text">
     <string>100</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="productInfoD">
    <property name="geometry">
     <rect>
      <x>215</x>
      <y>152</y>
      <width>120</width>
      <height>48</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Comic Sans MS&quot;;
border:1px solid #2ecc71;
background-color: rgb(44, 62, 80);
color: rgb(241, 196, 15);
</string>
    </property>
    <property name="text">
     <string>Product Info</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="productBrandD">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>200</y>
      <width>200</width>
      <height>48</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Comic Sans MS&quot;;
border:none;
border-left:1px solid #2ecc71;
border-top:1px solid #2ecc71;
background-color: rgb(44, 62, 80);
color: rgb(241, 196, 15);</string>
    </property>
    <property name="text">
     <string>Product Brand</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="productRatingD">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>248</y>
      <width>200</width>
      <height>46</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Comic Sans MS&quot;;
border:none;
border-left:1px solid #2ecc71;
border-top:1px solid #2ecc71;
background-color: rgb(44, 62, 80);
color: rgb(241, 196, 15);

</string>
    </property>
    <property name="text">
     <string>Product Rating</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="productRatingCountD">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>294</y>
      <width>200</width>
      <height>46</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Comic Sans MS&quot;;
border:none;
border-left:1px solid #2ecc71;
border-top:1px solid #2ecc71;
border-bottom:1px solid #2ecc71;
background-color: rgb(44, 62, 80);
color: rgb(241, 196, 15);
</string>
    </property>
    <property name="text">
     <string>Product Rating Count</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="productImageD">
    <property name="geometry">
     <rect>
      <x>335</x>
      <y>152</y>
      <width>140</width>
      <height>48</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Comic Sans MS&quot;;
border:none;
border-right:1px solid #2ecc71;
border-bottom:1px solid #2ecc71;
border-top:1px solid #2ecc71;
background-color: rgb(44, 62, 80);
color: rgb(241, 196, 15);
</string>
    </property>
    <property name="text">
     <string>Product Image</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="productFullNameD">
    <property name="geometry">
     <rect>
      <x>475</x>
      <y>152</y>
      <width>180</width>
      <height>48</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Comic Sans MS&quot;;
border:none;
border-right:1px solid #2ecc71;
border-bottom:1px solid #2ecc71;
border-top:1px solid #2ecc71;
background-color: rgb(44, 62, 80);
color: rgb(241, 196, 15);
</string>
    </property>
    <property name="text">
     <string>Product Full Name</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="productPriceD">
    <property name="geometry">
     <rect>
      <x>655</x>
      <y>152</y>
      <width>130</width>
      <height>48</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Comic Sans MS&quot;;
border:none;
border-right:1px solid #2ecc71;
border-bottom:1px solid #2ecc71;
border-top:1px solid #2ecc71;
background-color: rgb(44, 62, 80);
color: rgb(241, 196, 15);
</string>
    </property>
    <property name="text">
     <string>Product Price</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="productFullName">
    <property name="geometry">
     <rect>
      <x>475</x>
      <y>200</y>
      <width>180</width>
      <height>140</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Comic Sans MS&quot;;
border:none;
border-bottom:1px solid #2ecc71;
border-right:1px solid #2ecc71;
background-color: rgb(44, 62, 80);
color: rgb(241, 196, 15);
</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="productPrice">
    <property name="geometry">
     <rect>
      <x>655</x>
      <y>200</y>
      <width>130</width>
      <height>140</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <strikeout>false</strikeout>
      <stylestrategy>PreferDefault</stylestrategy>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Comic Sans MS&quot;;
border:none;
border-bottom:1px solid #2ecc71;
border-right:1px solid #2ecc71;
background-color: rgb(44, 62, 80);
color: rgb(241, 196, 15);
</string>
    </property>
    <property name="text">
     <string>3875.50</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="productUrlD">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>340</y>
      <width>200</width>
      <height>48</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Comic Sans MS&quot;;
border:none;
border-left:1px solid #2ecc71;
border-bottom:1px solid #2ecc71;
background-color: rgb(44, 62, 80);
color: rgb(241, 196, 15);

</string>
    </property>
    <property name="text">
     <string>Product Url</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="productUrl">
    <property name="geometry">
     <rect>
      <x>215</x>
      <y>340</y>
      <width>570</width>
      <height>48</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Comic Sans MS&quot;; 
border:none; border-left:1px solid #2ecc71; 
border-right:1px solid #2ecc71; 
border-bottom:1px solid #2ecc71; 
background-color: rgb(44, 62, 80); 
color: rgb(241, 196, 15);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="openExternalLinks">
     <bool>true</bool>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::LinksAccessibleByMouse</set>
    </property>
   </widget>
   <widget class="QPushButton" name="showPreviousButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>468</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>showPreviousProductButtonImage.png</normaloff>showPreviousProductButtonImage.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="showNextButton">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>468</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background:transparent;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>showNextProductButtonImage.png</normaloff>showNextProductButtonImage.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>325</x>
      <y>468</y>
      <width>150</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
font: 16pt &quot;Comic Sans MS&quot;;
color: rgb(44, 62, 80);
background-color: rgb(241, 196, 15);
border-radius:20px;
}

QPushButton::hover{
background-color: rgb(44, 62, 80);
color: rgb(241, 196, 15);

}
</string>
    </property>
    <property name="text">
     <string>Save Product</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>215</x>
      <y>0</y>
      <width>370</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;Comic Sans MS&quot;;
color: rgb(52, 73, 94);
background-color: rgb(241, 196, 15);
border:none;
border-bottom:1px solid #ecf0f1;
</string>
    </property>
    <property name="text">
     <string>n11 Scrapper</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="storeNameD">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>388</y>
      <width>200</width>
      <height>48</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Comic Sans MS&quot;;
border:none;
border-left:1px solid #2ecc71;
border-bottom:1px solid #2ecc71;
background-color: rgb(44, 62, 80);
color: rgb(241, 196, 15);

</string>
    </property>
    <property name="text">
     <string>Store Name</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="storeRatingD">
    <property name="geometry">
     <rect>
      <x>585</x>
      <y>388</y>
      <width>200</width>
      <height>48</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Comic Sans MS&quot;;
border:none;
border-left:1px solid #2ecc71;
border-bottom:1px solid #2ecc71;
border-right:1px solid #2ecc71;

background-color: rgb(44, 62, 80);
color: rgb(241, 196, 15);

</string>
    </property>
    <property name="text">
     <string>Store Rating</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="storeName">
    <property name="geometry">
     <rect>
      <x>215</x>
      <y>388</y>
      <width>185</width>
      <height>48</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Comic Sans MS&quot;;
border:none;
border-left:1px solid #2ecc71;
border-bottom:1px solid #2ecc71;
background-color: rgb(44, 62, 80);
color: rgb(241, 196, 15);


</string>
    </property>
    <property name="text">
     <string>temp</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
    </property>
    <property name="buddy">
     <cstring></cstring>
    </property>
   </widget>
   <widget class="QLabel" name="storeRating">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>388</y>
      <width>185</width>
      <height>48</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Comic Sans MS&quot;;
border:none;
border-left:1px solid #2ecc71;
border-bottom:1px solid #2ecc71;
background-color: rgb(44, 62, 80);
color: rgb(241, 196, 15);

</string>
    </property>
    <property name="text">
     <string>temp</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="categoryD">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>215</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Comic Sans MS&quot;;
border:none;
border-right:1px solid #ecf0f1;
border-bottom:1px solid #ecf0f1;
background-color: rgb(44, 62, 80);
color: rgb(241, 196, 15);</string>
    </property>
    <property name="text">
     <string>Category</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="totalProductD">
    <property name="geometry">
     <rect>
      <x>585</x>
      <y>0</y>
      <width>215</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Comic Sans MS&quot;;
border:none;
border-left:1px solid #ecf0f1;
border-bottom:1px solid #ecf0f1;
background-color: rgb(44, 62, 80);
color: rgb(241, 196, 15);</string>
    </property>
    <property name="text">
     <string>Total Product</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="category">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>215</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Comic Sans MS&quot;;
border:none;
border-bottom-right-radius: 25px;
border-bottom:1px solid #ecf0f1;
border-right:1px solid #ecf0f1;
color: rgb(52, 73, 94);
background-color: rgb(241, 196, 15);</string>
    </property>
    <property name="text">
     <string>temp</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="totalProduct">
    <property name="geometry">
     <rect>
      <x>585</x>
      <y>70</y>
      <width>215</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Comic Sans MS&quot;;
border:none;
border-bottom-left-radius: 25px;
border-left:1px solid #ecf0f1;
border-bottom:1px solid #ecf0f1;
color: rgb(52, 73, 94);
background-color: rgb(241, 196, 15);</string>
    </property>
    <property name="text">
     <string>temp</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
